test/loopback_sdk_angular_test.js

```diff
@@ -51,5 +51,14 @@ exports.loopback_angular = {
     test.equal(parse.moduleName(script), 'customServices');
     test.equal(parse.baseUrl(script), 'http://custom/api');
     test.done();
+  },
+  ignore_models_option: function(test) {
```

Comment:
Add an empty line between test cases to visually delimit them. See e.g. L47-L48.